### PR TITLE
Template update 2.10

### PR DESCRIPTION
**Pull request by danilodileo**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metatdenovo/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/metatdenovo _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

Based on template2.10 I updated the dev branch

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -17,8 +17,9 @@ nextflow.enable.dsl = 2
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-// Part of the template. We have no use for this?
-// params.fasta = WorkflowMain.getGenomeAttribute(params, 'fasta')
+//   This is an example of how to use getGenomeAttribute() to fetch parameters
+//   from igenomes.config using `--genome`
+//params.fasta = WorkflowMain.getGenomeAttribute(params, 'fasta')
```
Comment (erikrikarddaniel) [inline main.nf]: I think we can just delete these lines.
```suggestion
```

File: nextflow.config [hunk 2]
```diff
@@ -220,8 +221,8 @@ profiles {
     test_rnaspades      { includeConfig 'conf/test_rnaspades.config'    }
     gitpod {
         executor.name          = 'local'
-        executor.cpus          = 16
-        executor.memory        = 60.GB
+        executor.cpus          = 4
+        executor.memory        = 8.GB
```
Comment (erikrikarddaniel) [inline nextflow.config]: I think I changed this to get the test to pass, have you made sure you can run it with 4 cores and 8 GiB RAM? We will need to discuss what tests to include in the CI procedure as they should not request so much resources.

File: nextflow_schema.json [hunk 3]
```diff
@@ -429,7 +429,6 @@
                     "type": "boolean",
                     "description": "Display help text.",
                     "fa_icon": "fas fa-question-circle",
-                    "default": false,
```
Comment (erikrikarddaniel) [inline nextflow_schema.json:432]: Is this a template thing, i.e. that defaults for booleans are not explicit any more?

**Review by erikrikarddaniel — APPROVED** [review]
A few comments, otherwise fine!

File: nextflow.config (hunk 2 above)
Comment (danilodileo) [reply nextflow.config]: I updated it

File: nextflow_schema.json (hunk 3 above)
Comment (danilodileo) [reply nextflow_schema.json:432]: I don't know. does it make difference?

File: nextflow.config (hunk 2 above)
Comment (erikrikarddaniel) [reply nextflow.config]: And the test passes with the new values?

File: nextflow_schema.json (hunk 3 above)
Comment (erikrikarddaniel) [reply nextflow_schema.json:432]: No, I was just curious.

File: nextflow.config (hunk 2 above)
Comment (danilodileo) [reply nextflow.config]: yes. it works

**Pull request merged**